File: 06 - Java/09-06-00-EjercicioPropuesto-ValorAleatorio-ULP.java
import java.util.Random;

public class ProgramaPrincipal {

    public static void main(String[] args){
        // Generamos un valor aleatorio
        // Creamos un objeto random
        var aleatorio = new Random();
        //Generamos los valores entre 0 y 100 (no incluye el limite)
        int valorAleatorio = aleatorio.nextInt(100 + 1);
        System.out.println("Valor aleatorio (0 y 100): " + valorAleatorio);
        // Numero aleatorio entre 100 y 200
        int min = 100, max = 200;
        valorAleatorio = aleatorio.nextInt(min, max + 1);
        System.out.println("Valor aleatorio (100 y 200): " + valorAleatorio);
    }
}

File: 03 - C++/09-06-00-EjercicioPropuesto-ValorAleatorio-ULP.cpp
#include <iostream>
#include <ctime>
#include <cstdlib>

using namespace std;

int main(){  
	// Iniciamos el generador de numeros aleatorio
	srand(time(0)); 
	//Generamos los valores entre 1 y 100
	int valorAleatorio = (rand()%100 + 1);
	cout << "Valor aleatorio (1 y 100):" << valorAleatorio << endl;
	// Numero aleatorio entre 100 y 200
	int min = 100, max = 200;
	valorAleatorio = rand()%((max + 1) - min) + min;
	cout << "Valor altearorio (100 y 200): " << valorAleatorio;
	
	return 0;
}
	
